compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/cfa/FirCallsEffectAnalyzer.kt

```diff
@@ -11,12 +11,12 @@ import org.jetbrains.kotlin.contracts.description.EventOccurrencesRange
 import org.jetbrains.kotlin.fir.FirSession
```

Comment:
General note: please change `[FIR] ControlFlowAnalyserCheckers support` to some more specific message. The most valuable point here is context which was added to CFA checkers (IMHO).